### DISCUSSION TITLE
Nested Templates

### DISCUSSION THREAD

**Discussion by macedotavares** [SilentVoid13/Templater#375] [Ideas & Feature Requests]
Hi!

I've been using Templater for a while, now. I've read the docs and searched the obsidian forum, but I haven't found any reference to nested templates, so I assume that it's not implemented.

It'd be cool, though. We could build small individual blocks, and then combine them into higher-order templates. 

Does this make sense to you?

Cheers!

**Comment by htmljenn** [comment]
Perhaps I don't understand what you mean, but I use what I think of as nested templates by calling the `tp.file.include()` command. For example, I nest basic metadata in my files with the line:

```
<% tp.file.include("[[metadata-basic]]") %>
```

As long as the file `[[metadata-basic]]` exists in my vault, it works just fine.
